oar/core/notification.py

```diff
@@ -265,11 +265,15 @@ def get_user_id_by_email(self, email):
             str: slack user id
         """
         email = self.transform_email(email)
+
+        if in_memory_cache.get(email) is not None:
+            return "<@%s>" % in_memory_cache.get(email)
```

Comment:
@jhuttana this line has duplicate pattern declaration, suggest to declare local var `userid`, fill the var by if/else logic
```
userid = None
if email in cache_dict:
	userid = cache_dict.get(email)
else:
	// api call
  userid = resp["user"]["id"]
  cache_dict[email] = userid

return "<@%s>" % userid
```
it looks more efficient

Reply:
also, we can add debug level log here to indicate the contact is retrieved from cache. so if you execute the cmd with option `-v/--debug`, the logic can be traced.

Reply:
Sure